hoomd/util.py

```diff
@@ -265,44 +261,25 @@ def array_to_strings(value):
 
 
 def trigger_preprocessing(trigger):
-    if isinstance(trigger, int):
```

Comment:
Why are tehse and other similar methods here at not in `typeconverter.py`?

Reply:
Moved them